packages/mobile/src/components/overflow-menu-drawer/OverflowMenuDrawer.tsx

```diff
@@ -89,11 +88,8 @@ export const OverflowMenuDrawer = () => {
     }
   })
 
-  const { data: event } = useRemixContest(id, {
-    entityType: EventEntityTypeEnum.Track
-  })
-
-  const isRemixContest = !!event
+  const { data: remixContest } = useRemixContest(id)
```

Comment:
ok to remove the entityType arg? does that open up the possibility that we'd erroneously get a remix contest for a track but display it on a playlist page or something?

Reply:
useRemixContest sets it now, which makes sense right?